src/foreman_mcp_server/server.py

```diff
@@ -118,21 +125,29 @@ def main(
 
     mcp = FastMCP(name="Foreman MCP Server")
 
+    # Use default CA bundle location if none specified
+    if ca_bundle is None and verify_ssl:
+        default_ca_path = Path.cwd() / "ca.pem"
+        if default_ca_path.exists():
+            ca_bundle = str(default_ca_path)
+
     foreman_api = None
     if transport == "stdio":
+        # apypie expects certificate path as verify_ssl parameter, not ca_bundle
+        verify_ssl_param = ca_bundle if ca_bundle else verify_ssl
```

Comment:
We can move it away from `if` statement, so we can pass this as `verify_ssl` param to the middleware init below, so we don't need to have this line there as well.